src/any-type-field/any-type-field.component.html

```diff
@@ -1,12 +1,12 @@
 <div [ngSwitch]="schema.componentType">
   <div *ngSwitchCase="'table-list'">
-    <table-list-field [values]="value" [schema]="schema" [path]="path"></table-list-field>
+    <table-list-field [values]="value" [supportValues]="value" [schema]="schema" [path]="path"></table-list-field>
   </div>
   <div *ngSwitchCase="'complex-list'">
-    <complex-list-field [values]="value" [schema]="schema" [path]="path"></complex-list-field>
+    <complex-list-field [values]="value" [supportValues]="value" [schema]="schema" [path]="path"></complex-list-field>
```

Comment:
what's the value o the value? Could you please give me an example?

Reply:
It's an Immutable List that populates tables, it's easiest to show on following screenshots (value of 'value' and corresponding table:
<img width="365" alt="Screenshot 2023-08-09 at 14 17 48" src="https://github.com/inveniosoftware-contrib/ng2-json-editor/assets/55505399/4b566ec3-8641-4577-a27d-75778e324bac">
<img width="528" alt="Screenshot 2023-08-09 at 14 17 40" src="https://github.com/inveniosoftware-contrib/ng2-json-editor/assets/55505399/035928e8-8d05-49bc-a0fe-ca840ee5b966">